javascript/src/main/utils/RoomSlotTableHelpers.js

```diff
@@ -0,0 +1,15 @@
+// Adapted from StackOverflow:
+// https://stackoverflow.com/questions/13898423/javascript-convert-24-hour-time-of-day-string-to-12-hour-time-with-am-pm-and-no
+const toAMPMFormat = (time) => {
```

Comment:
Similar to formatTimeToLocalTime, this doesn't have to be refactored in this PR but we should consider a dedicated library for this in the long run.